example/flaskr/templates/base.html

```diff
@@ -5,6 +5,7 @@
   <h1><a href="{{ url_for('auth.arrival') }}">Ethical Eating</a></h1>
   <ul>
     {% if g.user %}
+      <li><a class="action" href="{{ url_for('cookbook.index') }}">My CookBook</a></li>
```

Comment:
Issue with saving recipe to cookbook:
![cookbook1](https://user-images.githubusercontent.com/44328786/101146339-ba000d00-35cf-11eb-9576-1d3ab63490ff.JPG)
When clicked, it now notifies user that it has been added
![cookbook2](https://user-images.githubusercontent.com/44328786/101146348-bc626700-35cf-11eb-9d4d-93484f78983f.JPG)
But going to index, it shows all recipes
![cookbook3](https://user-images.githubusercontent.com/44328786/101146360-bf5d5780-35cf-11eb-9c61-9c26eb20db2f.JPG)
I think there is a bug with the sql query in cookbook index but could not figure it out

Reply:
Not all displayed recipes are actually part of the recipebook. If you click other recipes in cookbook index, they can be added to the recipebook (by flash message). This makes me think that the sql query isn't filtering enough. I tried to change the join clause hoping this would work but it seem like a toughly.

Reply:
noted, filing this as an issue. Please count this towards your 3 issues for HW5.